packages/foreman/python-websockify/python-websockify.spec

```diff
@@ -1,38 +1,35 @@
-%global pkgname websockify
+# Created by pyp2rpm-3.3.8
```

Comment:
I explicitly copied it from EPEL to keep it as close as possible.

Reply:
The benefit I saw here is have defined a fairly standard way within our packaging of how Python packages should look for EL8+ with our Pulpcore packaging paradigms. Thus, having our python packages match this is more clear to developers where to look and what the patterns of packaging are. That this came from Fedora and is intended to match it "1:1" rather than our own is not obvious. 

However, I can put it back if there is a strong feeling in that direction and I'll add a comment at the top to make it clear.

Reply:
@ekohl could you weigh in one more time?

Reply:
I do feel that a manpage is good for something that places in `/usr/bin` and that isn't in the PyPI tarball, but is in the current RPM.